builder/lxc/config.go

```diff
@@ -29,6 +30,18 @@ type Config struct {
 	ctx interpolate.Context
 }
 
+type ExportConfig struct {
+	Filename string  `mapstructure:"filename"`
```

Comment:
I think you can remove the `mapstructure:"filename"` stuff if the struct property name matches the config property name.